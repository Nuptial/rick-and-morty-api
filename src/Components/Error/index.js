import {
    Grid,
    Typography
  } from "@mui/material";
import { Link } from "react-router-dom";

const Error = () => {
  return (
    <Grid
      container
      spacing={0}
      direction="column"
      alignItems="center"
      justifyContent="center"
      style={{ minHeight: "100vh" }}
    >
      <p>
        <Link to="/">Go to the home page</Link>
      </p>
      <Grid item xs={3}>
        <Typography>Error...</Typography>
      </Grid>
    </Grid>
  );
};

export default Error;
