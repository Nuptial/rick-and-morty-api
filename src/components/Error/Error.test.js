import renderer from 'react-test-renderer';
import { BrowserRouter } from "react-router-dom";
import Error from '../Error';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<BrowserRouter><Error /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});