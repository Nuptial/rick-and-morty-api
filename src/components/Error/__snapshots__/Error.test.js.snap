// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-l6hioc-MuiGrid-root"
  style={
    Object {
      "minHeight": "100vh",
    }
  }
>
  <p>
    <a
      href="/"
      onClick={[Function]}
    >
      Go to the home page
    </a>
  </p>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 css-1equabv-MuiGrid-root"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Error...
    </p>
  </div>
</div>
`;
